### PR TITLE
Update Critical-CH restart time in navigation params

### PR DESCRIPTION
**Pull request by arichiv**
This is needed for https://github.com/w3c/navigation-timing/pull/188/.

***
<a href="https://pr-preview.s3.amazonaws.com/WICG/client-hints-infrastructure/pull/153.html" title="Last updated on Jun 7, 2023, 1:27 PM UTC (edb2997)">Preview</a> | <a href="https://pr-preview.s3.amazonaws.com/WICG/client-hints-infrastructure/153/96a82df...edb2997.html" title="Last updated on Jun 7, 2023, 1:27 PM UTC (edb2997)">Diff</a>

### CODE REVIEW

File: index.bs [hunk 1]
```diff
@@ -294,10 +294,18 @@ At [=populating a session history entry=], in step 6 after substep 7 insert the
  <li>If |shouldReloadForCriticalClientHints| then:
  <ol>
    <li>Set <var>navigable</var>'s [=has reloaded for Critical-CH=] to `true`.
+   <li>Set <var>navigationParams</var>'s [=critical ch restart time=] to be the [=current high resolution time=].
    <li><a spec=HTML>reload</a> <var>navigable</var>.
```
Comment (yoavweiss) [inline index.bs:298]: Would reloading the navigable not restart its document load timing info? /cc @noamr

Comment (arichiv) [reply index.bs:297]: I think I fixed this by storing the time in the navigable until the next refresh

**Comment by arichiv** [comment]
@yoavweiss Have time to take a look?

File: index.bs [hunk 2]
```diff
@@ -286,18 +286,29 @@ Navigation response {#navigation-response}
...
  </ol>
+ <li>If <var>navigable</var>'s [=time reloaded for Critical-CH=] is not 0:
+ <ol>
+    <li>Set <var>navigationParams</var>'s [=critical ch restart time=] to be <var>navigable</var>'s [=time reloaded for Critical-CH=].
+ </ol>
 </ol>
 
+At [=navigation params=], append the following:
+<dl>
+<dt><dfn>critical ch restart time</dfn></dt>
```
Comment (noamr) [inline index.bs:307]: Where do you read this value?

Comment (noamr) [reply index.bs:307]: Yes! You should be explicit about where in HTML this is read, in this monkey-patch fashion. (In this and that section, pass this param to `create the navigation timing entry` or so. This way I can go to the link and check that the whole algorithm makes sense.

Comment (arichiv) [reply index.bs]: I think I'm already being as specific as the other code in this section? I'm adding it to a struct and specifying it'll be used in "creating the navigation timing entry", just like "[navigation timing type](https://html.spec.whatwg.org/multipage/browsing-the-web.html#navigation-params-nav-timing-type)".

I hear what you're saying about a missing link, but I'm not seeing a point where it's spelled out any more explicitly.

**Comment by noamr** [comment]
@yoavweiss I'm not a reviewer for this spec. Could you take a look?
IMO it's still quite handwavy and the new issue #154 is quite a big technical debt, leaving this spec in the explainer realm, but could use an extra pair of eyes.

File: index.bs [hunk 3]
```diff
@@ -277,27 +279,43 @@ Navigable {#navigable}
...
+ </ol>
+ <li>If <var>navigable</var>'s [=Critical-CH restart time=] is not 0:
  <ol>
-   <li>Set <var>navigable</var>'s [=has reloaded for Critical-CH=] to `true`.
-   <li><a spec=HTML>reload</a> <var>navigable</var>.
+    <li>Set <var>navigationParams</var>'s [=Critical-CH restart time=] to be <var>navigable</var>'s [=Critical-CH restart time=].
  </ol>
 </ol>
 
+Issue(154): Clarify how the restart logic integrates with the HTML spec.
```
Comment (yoavweiss) [inline index.bs:309]: s/the restart logic/"Restart the initial navigation"/

**Review by yoavweiss — APPROVED** [review]
LGTM % nit on the issue wording

**Pull request merged**